### PR TITLE
change alerts.query into alerts.assertion

### PR DESCRIPTION
**Pull request by lukman7788**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Change alerts.query into alerts.assertion(#877 )

## How did you implement / how did you fix it  
1.  change required field from `alert.query` into `alert.assertion`
2. add compability convertion for deprecated config

## How to test  
1. test success with old config(using `alert.query`)
image
image

2. test success with new config(using `alert.assertion`)
image

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -557,6 +559,31 @@ Please refer to the Monika documentations on how to how to configure notificatio
...
+              showMessage = true
+            }
+          })
+        }
+      })
+    }
+
+    if (showMessage) {
+      log.warn(
+        'Monika config "alerts.query" was Deprecated! please change into "alerts.assertion"'
```
Comment (nicnocquee) [inline src/index.ts:583]: "is deprecated. Please use `alerts.assertion`"

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (lukman7788) [reply src/index.ts]: warning message update

File: docs/src/pages/guides/alerts.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ probes:
       - method: GET
         url: https://github.com
         alerts:
-          - query: response.size >= 10000
+          - assertion: response.size >= 10000
```
Comment (nicnocquee) [inline docs/src/pages/guides/alerts.md:16]: Change the title of this section to Alert's Assertion

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: docs/src/pages/guides/alerts.md (hunk 2 above)
Comment (lukman7788) [reply docs/src/pages/guides/alerts.md:16]: docs was updated @nicnocquee 

Screen Shot 2022-09-14 at 13 32 48

Screen Shot 2022-09-14 at 13 33 09

**Review by nicnocquee — APPROVED** [review]

File: src/index.ts [hunk 3]
```diff
@@ -560,6 +563,47 @@ Please refer to the Monika documentations on how to how to configure notificatio
...
+              }
+              showMessage = true
+            }
+          }
+        }
+      }
+    }
+
+    if (showMessage) {
+      log.warn('is deprecated. Please use "alerts.assertion"')
```
Comment (dennypradipta) [inline src/index.ts:601]: Needs more context about what is being deprecated?

**Review by dennypradipta — APPROVED** [review]

**Pull request merged**